jetty-core/jetty-util/src/main/java/org/eclipse/jetty/util/ConcurrentPool.java

```diff
@@ -53,27 +54,20 @@ public class ConcurrentPool<P> implements Pool<P>, Dumpable
      */
     public static final int OPTIMAL_MAX_SIZE = 256;
 
+    private static final boolean STRONG_DEFAULT = Boolean.getBoolean(ConcurrentPool.class.getName() + ".STRONG_DEFAULT");
```

Comment:
I prefer explicit configuration over JVM system property.

Reply:
yep, this is just a temporary thing for testing.    I'll put a TODO in.   Ultimately we will need to pick what the default is and then have properties and modules and xml updated to allow configuration.